common-arcana.lic

```diff
@@ -150,6 +159,26 @@ module DRCA
     DRC.retreat(settings.ignored_npcs) unless spell['skip_retreat']
   end
 
+  def prepare_to_cast_runestone?(spell, settings)
```

Comment:
If the only thing you are using `settings` for is `runestone_storage`, I would prefer to just pass that in (same with `get_runestone?`).

Reply:
Okay.